client/static/index.html

```diff
@@ -85,15 +85,15 @@
 
                 <!-- Nav tabs in bottom pane -->
                 <nav class="nav nav-tabs mt-3 mb-3">
-                    <a class="nav-item nav-link active" id="code-tab" data-bs-toggle="tab" data-bs-target="#code" role="tab">Code</a>
-                    <a class="nav-item nav-link" id="log-tab" data-bs-toggle="tab" data-bs-target="#log" role="tab">Log</a>
-                    <a class="nav-item nav-link" id="player-tab" data-bs-toggle="tab" data-bs-target="#player" role="tab">Player</a>
-                    <a class="nav-item nav-link" id="news-tab" data-bs-toggle="tab" data-bs-target="#news" role="tab">News</a>
-                    <a class="nav-item nav-link" id="general-tab" data-bs-toggle="tab" data-bs-target="#general" role="tab">General</a>
-                    <a class="nav-item nav-link" id="api-tab" data-bs-toggle="tab" data-bs-target="#api" role="tab">Api</a>
-                    <a class="nav-item nav-link" id="levels-tab" data-bs-toggle="tab" data-bs-target="#levels" role="tab">Levels</a>
-                    <a class="nav-item nav-link" id="keybindings-tab" data-bs-toggle="tab" data-bs-target="#keybindings" role="tab">Shortcuts</a>
-                    <a class="nav-item nav-link" id="account-tab" data-bs-toggle="tab" data-bs-target="#account" role="tab">Account</a>
+                    <a class="nav-item nav-link active" id="code-tab" data-bs-toggle="tab" data-bs-target="#code" role="button">Code</a>
```

Comment:
I'm curious, what does this do? Looking again at the Bootstrap documentation, I think role="tab" is correct, but there's a suggestion that I should use <button> instead of <a>. Did you see something I didn't?

Reply:
Oh, now I get it. You're changing the cursor shape when you hover!
Changing the <a> to <button> seems to do the trick. This is recommended in the Bootstrap documentation:
https://getbootstrap.com/docs/5.1/components/navs-tabs/, under JavaScript behavior.